### PR TITLE
Add a bus alias

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

**Comment by XWB** [comment]
We still need to register `AsyncPersistPageHandler` because not everyone uses autoconfigure (bundles should not rely on this feature).

**Comment by deguif** [comment]
@XWB yes, that's done in your PR #1739 or I'm missing something?

**Comment by XWB** [comment]
Apologies, I assumed this PR was meant to replace the other one :)

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('messenger')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('message_bus')->defaultNull()->end()
+                        ->scalarNode('bus')->defaultNull()->end()
```
Comment (XWB) [inline src/DependencyInjection/Configuration.php:60]: I wonder, what's the point of `defaultNull` if we're setting a default (`messenger.default_bus`) in the extension?

Comment (deguif) [reply src/DependencyInjection/Configuration.php:60]: I'm not an expert with the configuration, that can probably be replaced by `defaultValue('messenger.default_bus')` here.

Comment (deguif) [reply src/DependencyInjection/Configuration.php]: I just replaced it

**Comment by darthf1** [comment]
@deguif Running on `dev-master`, I was preparing a PR which would continue on your work here and supersedes #1739.

```yaml
  fos_elastica.async_parsist_page_handler:
    class: FOS\ElasticaBundle\Message\Handler\AsyncPersistPageHandler
    arguments:
      - '@fos_elastica.async_pager_persister'
    tags: [{ name: messenger.message_handler, bus: fos_elastica.messenger.bus }]
    autoconfigure: false
```
The issue is that the following error is presented: 
```
Invalid handler service "fos_elastica.async_parsist_page_handler": bus "fos_elastica.messenger.bus" specified on the tag "messenger.message_handler" does not exist (known ones are: "command.bus", "event.bus", "query.bus").
```

Those "known" three busses are configured in my `messenger.yaml`: 
```yaml
framework:
  messenger:
    default_bus: command.bus
    buses:
      command.bus: ~
      event.bus: ~
      query.bus: ~
```
Is there additional work needed to make SF aware of this library bus / alias?

**Comment by deguif** [comment]
@darthf1 could you check the container to see if the alias `fos_elastica.messenger.bus` is effective, and was not removed from it?

**Comment by darthf1** [comment]
@deguif this is my ouput for `docker-compose exec php bin/console debug:container fos_elastica.messenger.bus`:
```
// This service is a private alias for the service debug.traced.command.bus                                            

Information for Service "debug.traced.command.bus"
==================================================

 ---------------- ------------------------------------------------- 
  Option           Value                                            
 ---------------- ------------------------------------------------- 
  Service ID       debug.traced.command.bus                         
  Class            Symfony\Component\Messenger\TraceableMessageBus  
  Tags             messenger.bus                                    
  Public           no                                               
  Synthetic        no                                               
  Lazy             no                                               
  Shared           yes                                              
  Abstract         no                                               
  Autowired        no                                               
  Autoconfigured   no                                               
 ---------------- ------------------------------------------------- 


 ! [NOTE] The "fos_elastica.messenger.bus" service or alias has been removed or inlined when the container was compiled.
```

**Comment by deguif** [comment]
@darthf1 could you try marking the alias `fos_elastica.messenger.bus` as public?

**Comment by darthf1** [comment]
@deguif that did not have an impact for me.

**Pull request merged**